jax/api.py

```diff
@@ -1575,6 +1574,56 @@ def _vjp(fun: lu.WrappedFun, *primals, **kwargs):
     return out_primal_py, vjp_py, tree_unflatten(aux_tree, aux)
 
 
+def linear_transpose(fun: Callable, *args) -> Callable:
+  """Transpose a function that is promised to be linear.
+
+  For linear functions, this transformation is equivalent to ``vjp``, but
+  avoids the overhead of overhead of computing the forward pass.
```

Comment:
FYI there's an error in the docstring :)

Reply:
good catch, fixed!